src/main/java/org/mamute/controllers/UserProfileController.java

```diff
@@ -199,6 +200,23 @@ public void uploadAvatar(UploadedFile avatar, @Load User user) {
 		result.use(json()).withoutRoot().from(attachment).serialize();
 	}
 
+	@CustomBrutauthRules(ModeratorOnlyRule.class)
+	@Delete
+	public void delete(Long userId) {
+		if (!this.environment.supports("deletable.users")) {
+			result.notFound();
+			return;
+		}
```

Comment:
We could use brutauth so we don't need to put this if every controller method with new disabled features